tofu/geom/_core_optics.py

```diff
@@ -2053,6 +2085,7 @@ def plot_focal_error_summed(
         ddist = np.linspace(dist_min, dist_max, int(ndist))
         di = np.linspace(di_min, di_max, int(ndi))
         error_lambda = np.full((di.size, ddist.size), np.nan)
+        test_lamb_interv = np.full((di.size, ddist.size), np.nan, dtype='bool')
```

Comment:
There is a contradiction here,
if you initialize the array with nan => type = float
But you want type = bool

=> initialize with False, or even shorter :
`test_lamb_interv = np.zeors((di.size, ddist.size), dtype='bool')`